pkgs/development/libraries/cdo/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchurl, curl, hdf5, netcdf
+, enable_cdi_lib ? false    # build, install and link to a CDI library [default=no]
+, enable_all_static ? false # build a completely statically linked CDO binary  [default=no]
+, enable_cxx ? false        # Use CXX as default compiler [default=no]
+, with_gnu_ld ? false       # assume the C compiler uses GNU ld [default=no]
```

Comment:
Is it actually necessary to specify `--with-gnu-ld`? I can't recall that ever being an issue, autoconf seems to be able to figure it out.

Also see other expressions that use feature flags; you normally use parameters named `fooSupport` or `withFoo`. For consistency, I recommend following that pattern.